open-grant-proposals/orbitdb-pinning/README.md

```diff
@@ -0,0 +1,194 @@
+# Open Grant Proposal: `OrbitDB Filecoin Integration`
+
+**Name of Project:** `OrbitDB Filecoin Integration`
+
+**Proposal Category:** `app-dev`
+
+**Proposer:** `@aphelionz` on behalf of `@eqlabs`
+
+**Do you agree to open source all work you do on behalf of this RFP and dual-license under MIT and
+APACHE2 licenses?:** Yes
+
+# Project Description
+
+[OrbitDB](https://github.com/orbitdb/orbit-db) is a distributed database engine that stores its
+data on IPFS as Conflict-free Replicated Data Types (CRDTs). OrbitDB has a number of prominent
+users including [Metamask](https://metamask.io), [bitcoin.com](https://bitcoin.com),
+[3Box](https://3box.io/), [AvionDB](https://dbdb.io/db/aviondb),
+and [BitcoinCash](https://www.bitcoincash.org/).
+
+Of all the features and improvements requested by our community, reliable pinning is perhaps the
+most common. An OrbitDB user wants their data to persist in the distributed swarm safely and
+reliably, with minimal configuration and attention required.
+
+If this grant is funded, we will deliver a baseline Filecoin integration into OrbitDB,
+allowing users to integrate to any complaint Filecoin node via the JSON RPC APIs.
+
+## Value
+
+At nearly [4,500 GitHub](https://github.com/orbitdb/orbit-db/stargazers) stars, OrbitDB is one of
+the more high-profile projects in the `js-ipfs` ecosystem. This means we have a track record of
+positive impacts that OrbitDB has already had on a prominent Protocol Labs project.
+
+The benefits of this have so far been:
+
+1. Raising awareness of IPFS through its transitive use in OrbitDB
+2. Driving improvements in IPFS due to community demand and response to OrbitDB
+
+It follows, then, that Filecoin could enjoy similar, synergistic effects with OrbitDB. The process of
```

Comment:
Looks like this thought gets cut off?

Reply:
Ah yes - just removed the sentence fragment there